### PR TITLE
Non-developer readme contribution

### PR DESCRIPTION
**Pull request by mxio**
I have written a clear, easy-to-comprehend introduction for Flint. I believe this introduction, free of complex technical terms, will swiftly and thoroughly inform anyone about the use-cases, merit, and advantages, including the celebrated features, of Flint. Seasoned developers, new programmers, and even nontechnical visitors will benefit from the clear language, exposing Flint to a broader range of users.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Check your project for common sources of contributor friction.
 
+#### For the Non-Developer
+
+Flint checks if your project's folder contains the proper files and structure to allow potential contributors to understand: 1) what the project's mission is, 2) how to contribute, 3) under what conditions the project can be used, and 4) how to install the project.
```
Comment (pengwynn) [inline README.md]: I like this. Two small suggestions. Could you swap:

> what the project's mission is

with

> the project's goals

... and 

> under what conditions the project can be used

with 

> usage guidelines

I think those changes make a clearer connection back to "understand" which introduces the series.

**Comment by pengwynn** [comment]
Thanks @mxio. I like this a lot. You can push up changes to this same branch and I'll apply the merge. :sparkling_heart:

**Comment by mxio** [comment]
Thanks! I've made the changes you noted.

**Comment by pengwynn** [comment]
:cake:

**Pull request merged**